core/server/main.cpp

```diff
@@ -147,6 +151,42 @@ int main(int a_argc, char **a_argv) {
 
         return 0;
       }
+      if (cfg_log_level != UINT_MAX) {
+        if (cfg_log_level >=
+            static_cast<unsigned int>(LogLevel::LAST_SENTINEL)) {
+          DL_ERROR(log_context,
```

Comment:
Should this throw an error instead of logging and defaulting?

Reply:
I think a default is fine, logs are not critical to the running of the service.